docs/adguard-for-android/solving-problems/compatibility-issues.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Known compatibility issues with Android apps
+sidebar_position: 14
+---
+
+## Conflict apps
+
+### Various VPN apps
+
+All VPN apps are incompatible with AdGuard running in VPN filtering mode since you cannot have two active VPNs at the same time. To solve this problem we can suggest you:
+
+* to use AdGuard VPN that has a compatibility mode with AdGuard out-of-the-box
+* to use AdGuard in Local HTTP Proxy mode. *Please note that 'manual' HTTP Proxy mode is broken since Android 10*
+* to figure out if the third-party VPN app supports SOCKS4/SOCKS5 mode. In this case you can configure a VPN app to work as outbound proxy and to set up local-host outbound proxy with params from the third-party app. AdGuard can be automatically configured with 'Orbot: Proxy with Tor'
+
+### Samsung Pay
+
+The Samsung Pay app does not work with AdGuard due to the presence of an active VPN interface. This issue is region-specific and all known issues were reported by users from South Korea. Find more details [here](https://github.com/AdguardTeam/AdguardForAndroid/issues/1209).
+
+### Private DNS
```

Comment:
Private DNS is not an 'app', we should move it in a separate section

Reply:
agree, renamed the above section as ## Conflicting apps and features